components/FriendPagePreview.js

```diff
@@ -56,47 +80,96 @@ const StyledNotice = styled.div`
 `;
 
 export default function FriendPagePreview({ friend }) {
+  const [value, setValue] = useState(false);
+
+  const handleClick = () => {
+    setValue(!value);
+  };
+
   return (
-    <StyledFriend>
-      <StyledNickname>{friend.nickname}</StyledNickname>
-      <StyledName>
-        {friend.first_name} {friend.last_name}
-      </StyledName>
-      <div>
-        <Image
-          src={friend.profileIconSource}
-          alt={friend.nickname}
-          width={180}
-          height={180}
-        />
-      </div>
+    <>
+      <StyledDiv>
+        <StyledWidth>
+          <StyledFriend>
+            <StyledNickname>{friend.nickname}</StyledNickname>
+            <StyledName>
+              {friend.first_name} {friend.last_name}
+            </StyledName>
+            <div>
+              <Image
+                src={friend.profileIconSource}
+                alt={friend.nickname}
+                width={180}
+                height={180}
+              />
+            </div>
+
+            <StyledContactList>
+              {friend.contactOptions &&
+                friend.contactOptions.map((options) =>
+                  hasContactOption(options) ? (
+                    <StyledList key={options.name}>
+                      <StyledButton href={options.number}>
+                        {options.name}
+                      </StyledButton>
+                    </StyledList>
+                  ) : (
+                    <></>
+                  )
```

Comment:
Please do not use an empty fragment as a `null` value. Use `null` instead.

```suggestion
                  ) : null
```

Reply:
ok